react/components/EXPERIMENTAL_TableTree/hooks/useCheckboxesContext.ts

```diff
@@ -0,0 +1,10 @@
+import { useContext } from 'react'
+
+import CheckboxesContext from '../checkboxContext'
+
+const useCheckboxesContext = () => {
```

Comment:
Is this file really needed?

Reply:
Not really, reduced to a single `contexts` file